pyproject.toml

```diff
@@ -102,7 +101,7 @@ test = [
     "dask[distributed]",
     "awkward>=2.3",
     "pyarrow",
-    "anndata[dask]",
+    "anndata[dask,lazy]",
```

Comment:
if we do this we don't need `test-full`

Reply:
Does this make the tests noticably slower? Otherwise why have `tests-full`?

Reply:
I mean, yes, there are more tests for sure (the entire `lazy` folder).  I separated it to ensure that we don't test our min dependencies against whatever `xarray` forces us to install.

Reply:
Well, I now changed it so local matches CI.

It was more reliably reproducible with the min-deps job, but it does affect the stable job too!